packages/components/pages/f-mfa/package.json

```diff
@@ -46,12 +46,19 @@
     "extends @justeat/browserslist-config-fozzie"
   ],
   "dependencies": {
-    "@justeat/f-services": "1.16.0"
+    "@justeat/f-services": "1.16.0",
+    "@justeat/f-vue-icons": "3.3.0",
```

Comment:
I _think_ we should be using `"@justeattakeaway/pie-icons-vue": "1.0.0"`?

Is this right, @ashleynolan?

Reply:
Yeh, it's fine to use f-vue-icons for those legacy images for now, until we work out how we want to serve those images and fully deprecate the old icons package (as these splash icons aren't in the new iconset)